src/services/data/data.service.ts

```diff
@@ -52,15 +37,17 @@ export class DataService implements OnDestroy {
           },
           name: {
             type: 'select',
-            defaultValue: name,
-            options: Object.keys(datasets),
+            defaultValue: defaultDatasetName,
```

Comment:
Why do we need to define the default for each one if it is always the same value?

Reply:
We need to make new meta object contain `fields name` and `enabled`.  It is going to replace the meta object of previous dataset.